src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/DynamicDebuggerProxy.cs

```diff
@@ -436,7 +442,20 @@ internal static object TryGetMemberValue(object obj, string name, bool ignoreExc
 
                 return result;
             }
-
+#if ENABLECOMBINDER
+            else if (obj != null && ComObjectType.IsAssignableFrom(obj.GetType()))
+            {
+                var comExclusionList = new string[] { "MailEnvelope" }; //add any com names to be excluded from dynamic view here
```

Comment:
>MailEnvelope [](start = 55, length = 12)

I wonder what is the history behind excluding `MailEnvelope` from the debug view and whether the exclusion is still relevant.

Reply:
That's all there is in Framework as far as comments go. Looks like it has been there since the sources were added / copied in 2010 (not sure how to go back further).

I tried removing the exclusion. And was able to see the `MailEnvelope` property for an Excel `Worksheet` and a Word `Document` just fine.